x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/Security.java

```diff
@@ -261,6 +263,8 @@
     private final SetOnce<ThreadContext> threadContext = new SetOnce<>();
     private final SetOnce<TokenService> tokenService = new SetOnce<>();
     private final SetOnce<SecurityActionFilter> securityActionFilter = new SetOnce<>();
+    private final SetOnce<SecurityIndexManager> securityIndex = new SetOnce<>();
```

Comment:
this is a nit and was there before this change, but maybe we should call this `securityIndexManager` everywhere instead of `securityIndex`

Reply:
I actually like `securityIndex`, since the methods all reflect the state of the index, not the manager itself.